src/models/User.js

```diff
@@ -0,0 +1,43 @@
+const users = (sequelize, DataTypes) => {
+  const User = sequelize.define('user', {
+    id: {
+      type: DataTypes.INTEGER,
+      unique: true,
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true
+    },
+    first_name: DataTypes.STRING,
+    last_name: DataTypes.STRING,
+    email: {
+      type: DataTypes.STRING,
```

Comment:
The user migration doesn't contain all fields such as email, password, and others. Please check.

Reply:
By the database structure that we are deliberating on, we are separating users profile from authentication details. the login table is the one that contains fields for 'email and password'.